monkestation/code/datums/keybinding/mob.dm

```diff
@@ -0,0 +1,37 @@
+/datum/keybinding/mob/item_pixel_shift
+	hotkey_keys = list("V")
+	name = "item_pixel_shift"
+	full_name = "Item Pixel Shift"
+	description = "Shift a pulled item's offset"
+	category = CATEGORY_MISC
+	keybind_signal = COMSIG_KB_MOB_ITEM_PIXEL_SHIFT_DOWN
+
+/datum/keybinding/mob/item_pixel_shift/down(client/user)
+	. = ..()
+	if(.)
+		return
+	user.mob.AddComponent(/datum/component/pixel_shift)
+	SEND_SIGNAL(user.mob, COMSIG_KB_MOB_ITEM_PIXEL_SHIFT_DOWN)
+
+/datum/keybinding/mob/item_pixel_shift/up(client/user)
+	. = ..()
+	SEND_SIGNAL(user.mob, COMSIG_KB_MOB_ITEM_PIXEL_SHIFT_UP)
+
+/datum/keybinding/mob/pixel_shift
+	hotkey_keys = list("B")
+	name = "pixel_shift"
+	full_name = "Pixel Shift"
+	description = "Shift your characters offset."
+	category = CATEGORY_MOVEMENT
+	keybind_signal = COMSIG_KB_MOB_PIXEL_SHIFT_DOWN
+
+/datum/keybinding/mob/pixel_shift/down(client/user)
+	. = ..()
+	if(.)
+		return
+	user.mob.AddComponent(/datum/component/pixel_shift)
+	SEND_SIGNAL(user.mob, COMSIG_KB_MOB_PIXEL_SHIFT_DOWN)
+
+/datum/keybinding/mob/pixel_shift/up(client/user)
+	. = ..()
+	SEND_SIGNAL(user.mob, COMSIG_KB_MOB_PIXEL_SHIFT_UP)
```

Comment:
i will merge this as soon as these keybinds are changed to living rather than mob